packages/block-library/src/gallery/gallery-image.native.js

```diff
@@ -0,0 +1,278 @@
+			isCropped, getStylesFromColorScheme } = this.props;
+
+		const { isUploadInProgress } = this.state;
+		const { isUploadFailed, retryMessage } = params;
+		const resizeMode = isCropped ? 'cover' : 'contain';
+
+		const imageStyle = [ style.image, { resizeMode },
+			isUploadInProgress ? style.imageUploading : undefined,
+		];
+
+		const overlayStyle = compose( style.overlay,
+			isSelected ? style.overlaySelected : undefined,
+		);
+
+		const captionPlaceholderStyle = getStylesFromColorScheme( style.captionPlaceholder, style.captionPlaceholderDark );
+
+		const shouldShowCaptionEditable = ! isUploadFailed && isSelected;
+		const shouldShowCaptionExpanded = ! isUploadFailed && ( ! isSelected && !! caption );
+
+		const captionContainerStyle = shouldShowCaptionExpanded ?
+			style.captionExpandedContainer :
+			style.captionContainer;
+
+		const captionStyle = shouldShowCaptionExpanded ?
+			style.captionExpanded :
+			getStylesFromColorScheme( style.caption, style.captionDark );
+
+		return (
+			<>
+				<Image
+					style={ imageStyle }
+					source={ { uri: url } }
+					// alt={ alt }
+					accessibilityLabel={ ariaLabel }
+					ref={ this.bindContainer }
+				/>
+				{ isUploadFailed && (
+					<View style={ style.uploadFailedContainer }>
+						<View style={ style.uploadFailed }>
+							<Icon icon={ 'warning' } { ...style.uploadFailedIcon } />
+						</View>
+						<Text style={ style.uploadFailedText }>{ retryMessage }</Text>
+					</View>
+				) }
+				<View style={ overlayStyle }>
+					{ ! isUploadInProgress && (
+					<>
+						{ isSelected && (
+							<View style={ style.toolbar }>
+								<View style={ style.moverButtonContainer } >
+									<Button
+										style={ buttonStyle }
+										icon="arrow-left-alt"
+										iconSize={ ICON_SIZE_ARROW }
+										onClick={ isFirstItem ? undefined : onMoveBackward }
+										accessibilityLabel={ __( 'Move Image Backward' ) }
+										aria-disabled={ isFirstItem }
+										disabled={ ! isSelected }
+									/>
+									<View style={ separatorStyle }></View>
+									<Button
+										style={ buttonStyle }
+										icon="arrow-right-alt"
+										iconSize={ ICON_SIZE_ARROW }
+										onClick={ isLastItem ? undefined : onMoveForward }
+										accessibilityLabel={ __( 'Move Image Forward' ) }
+										aria-disabled={ isLastItem }
+										disabled={ ! isSelected }
+									/>
+								</View>
+								<Button
+									style={ removeButtonStyle }
+									icon="no-alt"
+									iconSize={ ICON_SIZE_REMOVE }
+									onClick={ onRemove }
+									accessibilityLabel={ __( 'Remove Image' ) }
+									disabled={ ! isSelected }
+								/>
+							</View>
+						) }
+						{ ( shouldShowCaptionEditable || shouldShowCaptionExpanded ) && (
+							<View style={ captionContainerStyle }>
+								<ScrollView nestedScrollEnabled keyboardShouldPersistTaps="handled">
+									<RichText
+										tagName=""
```

Comment:
We'll need to apply [this fix](https://github.com/WordPress/gutenberg/pull/18818) to here also

Reply:
Strangely, I don't observe the reported behavior within the gallery-image captions when `tagName=""` (tested on both iOS and Android), but I _do_ see the Android regression in the _block-caption_ described here: https://github.com/WordPress/gutenberg/pull/18818#issuecomment-559818548. Also, when I add `tagName="p"` to the gallery-image, that regression also occurs for the gallery-image caption.

Were you able to reproduce the original regression [described here](https://github.com/wordpress-mobile/WordPress-iOS/issues/13022) within the gallery-image caption?

Reply:
It is observable only on iOS, and Caption needs to be empty:

<img width="188" alt="Screen Shot 2019-12-03 at 10 46 38" src="https://user-images.githubusercontent.com/5032900/70030607-a126db00-15ba-11ea-926d-6a25d5dfb70c.png">

I tried this just now in try/gallery-draft branch

Reply:
Maybe you are using an older version of develop branch on WPiOS, this is a regression of a recent change on AzteciOS. So you need to use latest develop to be able to see it.(And latest develop requires with XCode 11.2)

Reply:
I've reproduced this on latest WPiOS, thanks for the extra info. I've added a `Platform.select` to avoid the regression on Android side, while still keeping the alignment fix for iOS. :heavy_check_mark: